nbs/models.hint.ipynb

```diff
@@ -17,18 +17,27 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    from neuralforecast.utils import AirPassengers, AirPassengersPanel, AirPassengersStatic
</pre><p>This is not necessary </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks. Took out, unnecesary imports.